lnbits/extensions/boltcards/crud.py

```diff
@@ -82,6 +82,14 @@ async def get_card(card_id: str) -> Optional[Card]:
 
     return Card.parse_obj(card)
 
+async def get_card_by_uid(card_uid: str) -> Optional[Card]:
+    row = await db.fetchone("SELECT * FROM boltcards.cards WHERE uid = ?", (card_uid,))
```

Comment:
Is the select case insensitive?

Reply:
Card UIDs should be stored in the db as uppercase. So, I just pushed a change that uppercases the card UID when creating/saving/searching by it, making it case insensitive from a user's perspective.